patches/server/0732-Fix-incosistency-issue-with-empty-map-items-in-CB.patch

```diff
@@ -0,0 +1,39 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Shane Freeder <theboyetronic@gmail.com>
+Date: Sun, 1 Aug 2021 09:49:06 +0100
+Subject: [PATCH] Fix incosistency issue with empty map items in CB
+
+
+diff --git a/src/main/java/net/minecraft/world/item/MapItem.java b/src/main/java/net/minecraft/world/item/MapItem.java
+index 24700c7e6a32cc30c97dccc21c5f3e3e6b6438e5..0d613586664796ee4e1f149ed82a2f8cf1091085 100644
+--- a/src/main/java/net/minecraft/world/item/MapItem.java
++++ b/src/main/java/net/minecraft/world/item/MapItem.java
+@@ -14,6 +14,7 @@ import net.minecraft.network.chat.Component;
+ import net.minecraft.network.chat.TranslatableComponent;
+ import net.minecraft.network.protocol.Packet;
+ import net.minecraft.resources.ResourceKey;
++import net.minecraft.server.MinecraftServer;
```

Comment:
thonkeng

Reply:
oof, was tryna look at something else at the same time and that slipped in